x-pack/plugins/security_solution/cypress/e2e/exceptions/alerts_table_flow/rule_exceptions.cy.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { LOADING_INDICATOR } from '../../../screens/security_header';
+import { getNewRule } from '../../../objects/rule';
+import { ALERTS_COUNT, EMPTY_ALERT_TABLE } from '../../../screens/alerts';
+import { createRule } from '../../../tasks/api_calls/rules';
+import { goToRuleDetails } from '../../../tasks/alerts_detection_rules';
+import {
+  addExceptionFromFirstAlert,
+  goToClosedAlertsOnRuleDetailsPage,
+  goToOpenedAlertsOnRuleDetailsPage,
+} from '../../../tasks/alerts';
+import {
+  addExceptionEntryFieldValue,
+  addExceptionEntryFieldValueValue,
+  addExceptionEntryOperatorValue,
+  addExceptionFlyoutItemName,
+  selectBulkCloseAlerts,
+  submitNewExceptionItem,
+} from '../../../tasks/exceptions';
+import {
+  esArchiverLoad,
+  esArchiverUnload,
+  esArchiverResetKibana,
+} from '../../../tasks/es_archiver';
+import { login, visitWithoutDateRange } from '../../../tasks/login';
+import {
+  goToAlertsTab,
+  goToExceptionsTab,
+  removeException,
+  waitForTheRuleToBeExecuted,
+} from '../../../tasks/rule_details';
+
+import { DETECTIONS_RULE_MANAGEMENT_URL } from '../../../urls/navigation';
+import { postDataView, deleteAlertsAndRules } from '../../../tasks/common';
+import {
+  EXCEPTION_CARD_ITEM_AFFECTED_RULES,
+  EXCEPTION_CARD_ITEM_AFFECTED_RULES_MENU_ITEM,
+  EXCEPTION_ITEM_VIEWER_CONTAINER,
+  NO_EXCEPTIONS_EXIST_PROMPT,
+} from '../../../screens/exceptions';
+import { waitForAlertsToPopulate } from '../../../tasks/create_new_rule';
+
+describe('Rule Exceptions workflows from Alert', () => {
+  const NUMBER_OF_AUDITBEAT_EXCEPTIONS_ALERTS = '1 alert';
+  const ITEM_NAME = 'Sample Exception List Item';
+  const newRule = getNewRule();
+  before(() => {
+    esArchiverResetKibana();
```

Comment:
Please, check if the use of this method is extremely necessary. Thanks.

Reply:
Changed to `cleanKibana`